packages/ndla-audio-search/src/AudioComponent.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+/* eslint jsx-a11y/media-has-caption: 0 */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import BEMHelper from 'react-bem-helper';
+
+const classes = new BEMHelper({
+  name: 'audio-component',
```

Comment:
prefix c står for component, så ville heller hatt audio-preview eller noe sånt i stedet.